### ISSUE TITLE
Humand Readable date in influxDB register

### ISSUE THREAD

**Issue by dotromain** [savoirfairelinux/mod-influxdb#32]
Hi,

I'm not sure to completly understading what is happen to me but, maybe is not my fault.
In my InfluxDB, every entry is the DB got a date like this "1970-01-01T00:00:01.447302965Z"
Is this normal ? I've skipped something?

Thanks !

**Comment by nerocide** [comment]
Hi,

@dotromain i got the exact same result. 

The epoch timestamp looks to be set uncorrectly, some digits are missing (see results below).

Here are some example logs from influxd.

I send a write query with curl omitting EPOCH intentionally:

*[httpd] 2015/12/03 16:21:01 write body received by handler: cpu_load_short,direction=in,host=server01,region=us-west value=2.0*

*[http] 2015/12/03 16:21:01 ::1 - - [03/Dec/2015:16:21:01 +0100] POST /write?db=shinken HTTP/1.1 204 0 - curl/7.29.0 76464142-99d1-11e5-8001-000000000000 6.025081ms*

**JSON result (human readable& epoch)=> ["2015-12-03T15:21:01.619925637Z","in","server01","us-west",2]
[1449156061619925637,"in","server01","us-west",2]**

Now let's see a request send from influxdb module plugged on the broker:

*[httpd] 2015/12/03 16:21:13 write body received by handler: metric_rta,host_name=shinken critical=3000i,min=0i,unit="ms",value=0.021,warning=1000i 1449156069
metric_pl,host_name=shinken critical=100i,max=100i,min=0i,unit="%",value=0i,warning=100i 1449156069
HOST_STATE,host_name=shinken acknowledged=0i,last_check=1449156069i,last_state_change=1449138340.08,output="PING OK - Packet loss = 0%, RTA = 0.02 ms",state=0i,state_type="HARD" 1449156069*

*[http] 2015/12/03 16:21:13 ::1 - root [03/Dec/2015:16:21:13 +0100] POST /write?db=shinken HTTP/1.1 204 0 - python-requests/2.8.1 7d276781-99d1-11e5-8002-000000000000 111.81718ms*

**JSON result (human readable& epoch)=> 
["1974-07-05T11:47:24.702900257Z","in","server01","us-west",2]
[142256844702900257,"in","server01","us-west",2]**

I'll try to dig this issue asap.

nb: FYI because i have many datapoints, in the results timestamps epoch & translated for human doesn't correspond, it's normal. Those are just here to show the number of digits.

**Comment by nerocide** [comment]
Hi,
i changed the precision setting on the write command of the python client and it is now working fine.

355c355
<                 self.db.write_points(buffer)
---
>                 self.db.write_points(buffer,time_precision="s")

**Comment by dotromain** [comment]
Yeah ! Thank you, it works like a charm.
